js/samples/advanced-nodejs/public/js/helpers.js

```diff
@@ -18,12 +18,22 @@ function getImmersiveReaderTokenAsync() {
 }
 
 function getSubdomainAsync() {
-    return new Promise((resolve, reject) => {
+    return new Promise(function (resolve, reject) {
         $.ajax({
             url: '/subdomain',
             type: 'GET',
-            success: subdomain => { resolve(subdomain); },
-            error: err => { reject(err); }
+            success: function (subdomain) { resolve(subdomain); },
+            error: function (err) { reject(err); }
+        });
+    });
+}
+
+function launchImmersiveReader(data, options) {
```

Comment:
Let's add comments in the sample html files to refer to this file to see how launchImmersiveReader is calling ImmersiveReader.launchAsync and getting tokens/subdomains

Reply:
Added comments